src/OrchardCore.Modules/OrchardCore.Facebook/Migrations/Widgets/migration.recipe.json

```diff
@@ -16,29 +16,29 @@
           "Name": "FacebookComments",
           "DisplayName": "Facebook Comments",
           "Settings": {
-            "Creatable": "False",
-            "Draftable": "False",
-            "Versionable": "True",
-            "Listable": "False",
-            "Securable": "True",
-            "Stereotype": "Widget"
+            "ContentTypeSettings": {
+              "Versionable": true,
+              "Securable": true,
+              "Stereotype": "Widget"
+            }
           },
```

Comment:
The json ignore null value serializer ignores anything that is `false` (or it's default value)